Superstitious/superstitious.html

```diff
@@ -0,0 +1,654 @@
+<div class='sheet-2colrow'>
+    <div class='sheet-col'>
+        <h4 class="section-head">Charakter Informationen</h4>
+        <div class="section">
+            <table style="width:100%">
+                <tr>
+                    <td>Name </td>
+                    <td colspan="3"><input type="text" name="attr_name" style="width:100%"></td>
+                </tr>
+                <tr>
+                    <td>Gespielt von </td>
+                    <td colspan="3"><input type="text" name="attr_player" style="width:100%"></td>
+                </tr>
+                <tr>
+                    <td>Typ </td>
+                    <td colspan="3"><input type="text" name="attr_typ" style="width:100%"></td>
+                </tr>
+                <tr>
+                                        <td>Geburtstag</td>
+                                        <td><input type="text" name="attr_birthdate" style="width:100%"></td>
+                    <td>Alter </td>
+                    <td><input type="text" name="attr_age" style="width:100%"></td>
+                                </tr>
+                                <tr>
+                    <td>Geschlecht </td>
+                    <td colspan=3><input type="text" name="attr_sex" style="width:100%"></td>
+                </tr>
+                <tr>
+                    <td>Wohnort </td>
+                    <td colspan="3"><input type="text" name="attr_residence" style="width:100%"></td>
+                </tr>
+                <tr>
+                    <td>Geburtsort </td>
+                    <td colspan="3"><input type="text" name="attr_birthplace" style="width:100%"></td>
+                </tr>
+            </table>
+        </div>
+    </div>
+    <div class='sheet-col'>
+        <h4 class="section-head">Attribute</h4>
+        <div class="section">
+            <table style="width:100%">

```

Comment:
Greetings,

All new character sheets to the repository are required to meet [Building Character Sheet standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository). We no longer accept sheet that use `<table>` as a layout design. It is highly recommended to use [grid](https://css-tricks.com/snippets/css/complete-guide-grid/) or [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for layout.